model-server/src/main/kotlin/org/modelix/model/server/handlers/ui/ContentExplorer.kt

```diff
@@ -188,6 +208,11 @@ class ContentExplorer(private val client: IModelClient, private val repoManager:
         }
     }
 
+    private fun getAncestorsAndSelf(expandTo: Long, tree: CLTree): Set<String> {
+        val seq = generateSequence(expandTo) { id -> tree.resolveElement(id)?.parentId }
```

Comment:
Non existing nodes, throw internal server errors.
`{"type":"/problems/unclassified-internal-server-error","title":"Internal server error","status":500,"detail":"Node doesn't exist: d6a (3434)"}`

Reply:
I added a new `NodeNotFoundException` with `HttpStatusCode.NotFound`

Reply:
Could you also add a test for that case?

Reply:
done